tslint.json

```diff
@@ -0,0 +1,48 @@
+{
+  "extends": [
+    "tslint:recommended",
```

Comment:
basically follow the `tslint:recommended` options, but ignoring some rules being unrelated with the code quality itself.

ignoring rules examples:

- alphabetic order
- format (parenthesis, spaces, etc)
- alignment